html/modules/custom/reliefweb_job_tagger/src/Plugin/QueueWorker/OchaAiJobTagTaggerWorker.php

```diff
@@ -167,6 +171,8 @@ public function processItem($data) {
         ->tag($text, [OchaAiTagTagger::CALCULATION_METHOD_MEAN_WITH_CUTOFF], OchaAiTagTagger::AVERAGE_FULL_AVERAGE);
     }
     catch (\Exception $exception) {
+      $this->setJobStatus($node, 'AI tagging failed, AI skipped', FALSE);
```

Comment:
Below there are 2 other `return;` (line 185 and line 190) when the `$data` is empty or doesn't contain the expected data. Shouldn't the job tagging status be updated in those cases as well?